web-app/js/portal/lang/en.js

```diff
@@ -228,6 +226,7 @@ OpenLayers.Lang.en = OpenLayers.Util.extend(OpenLayers.Lang.en, {
     depthLabel: 'Depth:',
     elevationLabel: 'Elevation:',
 
+    showHideTooltip: '{0} from the map',
     removeDataCollectionTooltip: 'Remove this data collection from the Subset and Download pages, the collection can be re-added from the search',
```

Comment:
Yes i'm happy because they are short and plain non techy/sciencey 
Have you got other ideas?